error/tpm2_error.c

```diff
@@ -0,0 +1,291 @@
+#include "../error/tpm2_error.h"
```

Comment:
should be in local defines block, no relative path (dang you eclipse)
Also needs copyright block